d2-schemas/src/main/pegasus/com/linkedin/d2/DarkClusterConfig.pdl

```diff
@@ -19,4 +19,14 @@ record DarkClusterConfig {
    * Max rate dispatcher can send to dark canary. Measured in qps. Will act as upper bound to protect canaries in case of traffic spikes
    */
   dispatcherOutboundMaxRate: int = 2147483647
-}
```

Comment:
removing would be backwards incompatible, and I think we can implement it sooner rather than later. What's the cleaner approach? I don't think those two are bound to the current constant qps implementation.

Reply:
we discussed offline. We will leave it as is, and consider deprecating it later if not needed. To remove it would require a major version bump of pegasus, which is undesirable if it can be avoided.